src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -84,7 +84,20 @@ We also have [template consent materials](https://docs.google.com/document/d/1il
 
 In fully confidential research, our client does not learn the identity of our participants, has no access to research notes, recordings or other data, and is not involved in analysis. They receive only fully anonymised findings.
 
-## Sharing your consent materials and process
+## Making consent materials inclusive and accessible
+
+We make sure our consent materials are inclusive and accessible, so they don’t create a barrier to participation.
+
+We use accessible formats, like text emails, HTML and Tagged PDF. And we create alternative materials,
+[like easy read](https://www.dxw.com/2019/08/including-people-with-learning-disabilities-in-research-you-dont-need-an-expensive-specialist-agency/), when needed. 
+
+We use [inclusive language](?). And we make it clear that we encourage participants with a range of needs to take part in research sessions,
```

Comment:
When I check the preview, inclusive language is a link to this same page itself, is this temporary until we add a section on inclusive language to the playbook or is this an oversight?

Reply:
It's temporary. It should link to the inclusive language material, when we've added it.

Reply:
I've removed it for now. We can add it back when the content on inclusive language has a home.